cmake/common.cmake

```diff
@@ -559,6 +565,17 @@ function(add_client_exe name)
 
 endfunction()
 
+function(add_virtual_only name)
```

Comment:
I am not certain we need this function. Instead, we could add some logic to `CCF_NETWORK_TEST_ARGS` to include `--enclave-type=virtual` if `VIRTUAL_ONLY` is `ON`. This would remove the need to add `add_virtual_only` for every unit test.

Reply:
Yes, you're absolutely right!